3ds Max/Max2Babylon/Exporter/AnimationGroup.cs

```diff
@@ -256,6 +256,19 @@ public class AnimationGroupList : List<AnimationGroup>
     {
         const string s_AnimationListPropertyName = "babylonjs_AnimationList";
 
+        public AnimationGroup GetAnimationGroupByName(string name)
```

Comment:
You can usually do all this with a single LINQ statement:

animationGroupList.Where(animationGroup => animationGroup.name = name);

since List implements IEnumerable